code/association_scan/TensorQTL/TensorQTL.ipynb

```diff
@@ -565,18 +564,21 @@
     "    import os\n",
     "    molecular_pheno_files = pd.read_csv(phenotype_file, sep = \"\\t\")\n",
     "    if \"#chr\" in molecular_pheno_files.columns:\n",
-    "        molecular_pheno_files = molecular_pheno_files.groupby(['#chr', '#dir']).size().reset_index(name='count').drop(\"count\",axis = 1).rename(columns = {\"#chr\":\"#id\"})\n",
+    "        molecular_pheno_files = molecular_pheno_files.groupby(['#chr','path']).size().reset_index(name='count').drop(\"count\",axis = 1).rename(columns = {\"#chr\":\"#id\"})\n",
+    "        print(molecular_pheno_files.head())\n",
     "    genotype_files = pd.read_csv(genotype_file,sep = \"\\t\")\n",
     "    genotype_files[\"#id\"] = [x.replace(\"chr\",\"\") for x in genotype_files[\"#id\"].astype(str)] # e.g. remove chr1 to 1\n",
     "    genotype_files[\"#path\"] = genotype_files[\"#path\"].apply(lambda x: adapt_file_path(x, genotype_file))\n",
     "    molecular_pheno_files[\"#id\"] = [x.replace(\"chr\",\"\") for x in molecular_pheno_files[\"#id\"].astype(str)]\n",
     "    input_files = molecular_pheno_files.merge(genotype_files, on = \"#id\")\n",
+    "\n",
     "    # Only keep chromosome specified in --chromosome\n",
     "    if len(chromosome) > 0:\n",
     "        input_files = input_files[input_files['#id'].isin(chromosome)]\n",
     "    input_files = input_files.values.tolist()\n",
     "    input_chroms = [x[0] for x in input_files]\n",
-    "    input_files = [x[1:] for x in input_files]"
+    "    print(input_files)\n",
```

Comment:
again, if you want to keep track of messages you should use formal logging (`logging` module in Python) not just `print`. Pleease clean it up